packages/components/src/Components/TagCount/TagCount.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './tagCount.scss';
+import TagModal from '../TagModal/TagModal'
+
+export default class TagCount extends React.Component
```

Comment:
Doesn't really have to be a class component, you should accept props so no need to store anything in state.